tox.ini

```diff
@@ -13,6 +13,7 @@ install_command =
 deps =
   pytest
   pytest-xdist
+  pytest-mock
```

Comment:
If it comes to rationale for the change around mocks... this is for the sake of making the tests more consistent with pytest conventions right? Or does it have some other positive side effects?

Reply:
In short, `pytest-mock` allows to write very concise tests and reduces a lot of boilerplate.
It also provides the very useful `mocker.spy()` and `mocker.stub()`  features, something doable with `unittest.mock` but requires lots of code. For more info read [pytest-mock documentation](https://pytest-mock.readthedocs.io/en/latest/)